src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -198,6 +198,10 @@ public sealed partial class CoClassAttribute : System.Attribute
         public CoClassAttribute(System.Type coClass) { }
         public System.Type CoClass { get { throw null; } }
     }
+    public static partial class CollectionsMarshal
+    {
+        public static System.Span<T> AsSpan<T>(System.Collections.Generic.List<T> list) { throw null; }
```

Comment:
This also needs to be annotated with nullability, `List<T>?`

Reply:
Yep, we should reflect the recent change made in the implementation within the ref assembly: https://github.com/dotnet/coreclr/pull/26903